.github/workflows/build.yml

```
@@ -90,28 +85,49 @@ jobs:
         shell: bash
         run: make ${{ env.FLAGS }} wheel
       - uses: actions/upload-artifact@v4
+        if: matrix.os != 'ubuntu'
         with:
           name: pip-wheel-${{ matrix.os }}-${{ matrix.python }}
           path: dist/*
           if-no-files-found: error
+      - name: Building manylinux package
+        if: matrix.os == 'ubuntu'
+        shell: bash
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2046:warning:10:15: Quote this to prevent word splitting

(shellcheck)